pkg/pod/entrypoint.go

```diff
@@ -172,6 +184,10 @@ func orderContainers(commonExtraEntrypointArgs []string, steps []corev1.Containe
 			}
 		}
 
+		if enableCancelUsingEntrypoint {
+			argsForEntrypoint = append(argsForEntrypoint, "-stop_on_cancel")
+		}
+
```

Comment:
Is it necessary to add this as an entrypoint arg, or can the entrypoint just listen for this file by default? The taskrun controller is already canceling in different ways based on the value of the feature flag.

Reply:
Good Idea! i removed the `stop_on_cancel`. feature flag and taskrun controller are enough to cancel tasks in different ways